src/org/zaproxy/zap/extension/ascanrulesBeta/HPP.java

```diff
@@ -182,6 +182,10 @@ public void scan() {
 			if(vulnLinks.isEmpty()) {
 				log.debug("Page not vulnerable to HPP attacks");
 			}
+		} catch (URIException e) {
+			if (log.isDebugEnabled()) {
+				log.debug("Failed to use the 'attack' URI: " + e.getMessage());
```

Comment:
As mentioned in the other pull request:
Yes, although we might not have it at this point (e.g. it might be a URI of a redirection), I will tweak the message. I will also change core to include the (actual) URI in the exception's message.